src/app/onboarding/member/page.tsx

```diff
@@ -48,21 +48,25 @@ const MembershipType = () => {
     }
   };
 
+  const toggleMemberSelection = (selected) => {
+    setMember(member === selected ? null : selected);
+  };
```

Comment:
there's a typescript error here because `selected` needs to be typed, and also the `setMember` function expects boolean  values only. You may need to refactor the OnboardingDetailsStore types to let `setMember` expect `null` types as well.